contrib/syslog-debun

```diff
@@ -31,25 +47,34 @@ pseao="ps -eao"
 dfk="df -k"
 dfh="df -h"
 duks="du -ks"
+grepq="grep -q"
+lddcmd="ldd"
 mount=mount
 varlimit=1000
 myplimit () { echo "Plimit query is not supported on this platform" >&3 ; }
 distpkgoffile () { echo "Package file search is not supported (yet) on this platform" >&3 ; }
 distpkgstatus () { echo "Package status query is not (yet) supported on this platform" >&3 ; }
 selftar="tar cf - ."
+gzipcmd="gzip -9"
 showdep="dpkg -S"
 pcapifparm=-i
 w=w
 vmstat=vmstat
 dmesg=dmesg
-tcpdump="tcpdump -p -s 1500 -w"
-getsyslogpids="pidof syslog-ng"
+tcpdumpcmd="tcpdump"
+tcpdumpopts="-p -s 1500 -w"
+getsyslogpids () { pidof syslog-ng ; }
 trace="strace -f"
-service_stop="/etc/init.d/syslog-ng stop"
-service_start="/etc/init.d/syslog-ng start"
-service_status="/etc/init.d/syslog-ng status"
-checkpid () { [ -d /proc/"$1" ]; }
-mywait () { jobs -p >${tmpdir}/sdn.jobs ; for i in `grep -v "\<$tailpid\>" ${tmpdir}/sdn.jobs` ; do wait $i ; done ; }
+initfile="/etc/init.d/syslog-ng"
+service_stop="${initfile} stop"
+service_start="${initfile} start"
+service_status="${initfile} status"
+#checkpid () { [ -d /proc/"$1" ]; }
+checkpid () { [ -n "$1" ] && kill -0 "$1" 2>/dev/null ; }
+#mywait () { jobs -p >${tmpdir}/sdn.jobs ; for i in `grep -v "\<$tailpid\>" ${tmpdir}/sdn.jobs` ; do wait $i ; done ; }
+mywait () { jobs -p >${tmpdir}/sdn.jobs ; for i in `grep -v "^$tailpid\$" ${tmpdir}/sdn.jobs` ; do wait $i ; done ; }
+is_available () { which "$1" >/dev/null 2>&1; }
```

Comment:
Or maybe the hash was the other solution what I used.
You have to check posix on that. I'm pretty sure, I did the research, just It was so handy at the time when I wrote that, that I haven't thought about documenting that research result into comments, I've just used them more or less consequently.
I suggest you the same: Do again the reseach.
As a starting point, use this link: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/hash.html

Summary: avoid wiches! ;-)